src/client/utils.ts

```diff
@@ -45,6 +47,10 @@ export function getJoinOptions(): IJoinRoomOpts {
     throw new Error('Invalid option --mode, values are training, match or spectate');
   }
 
+  if (typeof config.shufflePlayers === 'boolean') {
```

Comment:
We could also add shufflePlayers: false by default in config.json

Reply:
We could, but I feel this is better if its an optional config. We should have the config file only contain crucial settings.